doc/manual/src/contributing/hacking.md

```diff
@@ -83,7 +83,7 @@ by:
 $ nix develop
 ```
 
-## Testing
+## Testing Nix
 
 Nix comes with three different flavors of tests: unit, functional and integration.
 
```

Comment:
This line is invalidated by your addition, and I see no point in maintaining it.

```suggestion
```